src/domain/Contact.ts

```diff
@@ -638,6 +650,28 @@ export default class Contact {
     });
   }
 
+  static parseManyPhonebook(response: PhonebookResponse[], source: DirectorySource): Array<Contact> {
+    return response.map(r => Contact.parsePhonebook(r, source));
+  }
+
+  static parsePhonebook(single: PhonebookResponse, source: DirectorySource): Contact {
+    return new Contact({
+      id: single.id,
+      sourceId: single.id,
+      name: `${single.firstname}${single.lastname ? ` ${single.lastname}` : ''}`,
+      email: single.email,
+      emails: [{ label: 'primary', email: single.email }],
+      number: single.number,
+      numbers: [{ label: 'primary', number: single.number }],
+      address: single.address,
+      birthday: single.birthday,
+      entreprise: single.enterprise,
+      note: single.note,
+      source: source.name,
+      backend: 'phonebook',
```

Comment:
There's no schema at the moment, but some of the fields are expected to be optional (e.g `birthday`, `enterprise`, `note`, `email/emails`). Should we provide a fallback for those values ?  ( `|| ''` )